src/components/Map/LeafletMap.js

```diff
@@ -1,83 +1,97 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useMemo, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import { makeStyles } from '@mui/styles';
-import { MapContainer, TileLayer} from 'react-leaflet';
+import { MapContainer, TileLayer } from 'react-leaflet';
 import { mapConfig } from '../../configuration/config';
 import { BasicSelect } from './basicSelect';
-import { useSelector, useDispatch } from 'react-redux'
 import { changeRegionValue } from '../../reducers/regionSelectSlice';
-import { changeZoom, changeCenter } from '../../reducers/mapPropertiesSlice'
-
+import { changeZoom, changeCenter } from '../../reducers/mapPropertiesSlice';
 
 const useStyles = makeStyles((theme) => ({
   leafletContainer: {
     height: '600px',
-    width:'60%'
+    width: '60%'
   }
 }));
 
 const regions = mapConfig.regions;
 
-/*function RegionSelect({ map }) {
-  
-  const handleRegionChange = (e) => {
-    map.setView(regions[e.target.value].mapProperties.center, regions[e.target.value].mapProperties.zoom)
-  }
-  return (
-    <select onChange={e => handleRegionChange(e)}>
+/*
+  function RegionSelect({ map }) {
+    const handleRegionChange = (e) => {
+      map.setView(
+                  regions[e.target.value].mapProperties.center,
+                  regions[e.target.value].mapProperties.zoom
+      );
+    }
+    return (
+      <select onChange={e => handleRegionChange(e)}>
       {
-        regions.map((element, key) => <option key={key} value={key}>{element.label}</option>)
+      regions.map((element, key) => <option key={key} value={key}>{element.label}</option>)
       };
-    </select>
-  )
-}*/
+      </select>
+    );
+  }
+*/
+
+// selector named functions for lint rules makes it easier to re-ruse if needed.
+const selectedRegionSelector = (state) => state.selectedRegion.value;
+const selecteZoomSelector = (state) => state.mapProperties.zoom;
+const selecteCenterSelector = (state) => state.mapProperties.center;
```

Comment:
typo in lines 39+40 and future propogations, const is named 'selecte...' instead of 'selected...'
Also typo in comment in line 37, 're-ruse', instead of 're-use'

Reply:
cool same issue in `MapCard.js`